pkg/ipamd/ipamd.go

```diff
@@ -552,6 +538,19 @@ func (c *IPAMContext) nodeInit() error {
 		vpcV4CIDRs = c.updateCIDRsRulesOnChange(vpcV4CIDRs)
 	}, 30*time.Second)
 
+	// Retrieve security groups
```

Comment:
We could have moved the logic after `nodeInit` is successful but should be fine since if nodeInit fails then aws-node will be in crash loop..Can you update the comment since it has a tight dependency on unmanaged ENIs and ENIs on multicard interfaces?

Reply:
We can't move it after `nodeInit` is successful as the first call to `RefreshSGIDs` populates the ENI cache, and that cache must be populated before we attach any new ENIs

Reply:
I'll update the comment to mention the dependency

Reply:
Updated in latest commit